server/pkg/property/value_typography.go

```diff
@@ -128,7 +128,8 @@ func (p *typePropertyTypography) String(i interface{}) string {
 	if !p.Validate(i) {
 		return ""
 	}
-	panic("not implemented!")
+	return ""
+	// Should be implemented if needed
```

Comment:
I think it is not required to implement because it is hard to represent Typography and Camera in CSV as a string.